server/src/app/views.py

```diff
@@ -72,6 +76,14 @@ def authorizationCallback():
         configSettings = getConfigSettings(myEnv)
 
         # this gets the token from Medicare.gov once the 'user' authenticates their Medicare.gov account
+        print('code')
```

Comment:
We'll probably want to remove these before we merge this in

Reply:
This has been completed